resizeobserver/index.html

```diff
@@ -0,0 +1,52 @@
+---
+feature_name: ResizeObserver
+chrome_version: 54
+feature_id: 5705346022637568
+---
+
+<h3>Background</h3>
+<p>
+  The <a href="https://wicg.github.io/ResizeObserver/">ResizeObserver API</a>
+  provides an API to get notified whenever an element changes its size.
+</p>
+<p>
+  This sample illustrates how to use <code>ResizeObserver</code> to implement
+  media query-like behavior on a per-element basis.
+<p>
+  The <a href="https://developers.google.com/web/updates/2016/10/resizeobserver"><em>Google Developers Web Updates</em></a>
+  article provides more details on <code>ResizeObserver</code>.
+</p>
+
+{% capture initial_output_content %}
+<p>
+  On browsers that support <code>ResizeObserver</code>, you should see a border
+  appear around the green box when resizing the window.
```

Comment:
For folks who have difficulty distinguishing colors, could you rephrase as something like "the third box" rather than "the green box"?

Also, any particular reason why there are six boxes? It seems like you could make your point with just two boxes—one of which is associated with an RO, and the other which isn't.